settings.gradle

```diff
@@ -25,4 +25,6 @@ include 'metadata-models-ext'
 include 'metadata-models-generator'
 include 'metadata-testing:metadata-models-test-utils'
 include 'metadata-testing:metadata-test-utils'
-include 'metadata-utils'
\ No newline at end of file
+include 'metadata-utils'
+include 'datahub-gms-graphql-api'
```

Comment:
Can we consider alternative names for this module? When I read 'api' in a Java project I tend to think of a module containing public interfaces / the publicly accessible portions of a library.

Basically we should try to capture the idea that the module contains a deployable service using the name. 

Alternatives could be 

- datahub-gms-graphql-spring-boot
- datahub-gms-graphql-service
- datahub-gms-graphql-server

Or something similar. Interested to hear your thoughts!

Reply:
`datahub-gms-graphql-service` makes sense to me.

Reply:
On second thought considering to rename it as just `datahub-gms-graphql`. Any thoughts?

Reply:
I'm concerned people will wonder what the difference is between `datahub-gms-graphql` and `datahub-gms-graphql-core` if there is no additional qualifier. 

I think it's important to include something in the name suggesting "this is just a graphql-compatible deployable"